apps/editing-toolkit/bin/sync-newspack-blocks.sh

```diff
@@ -134,7 +134,7 @@ echo -n "phpcbf: "
 
 if [ "$PHPCBF_ERRORED" = 1 ] ; then
 	echo '!! There was an error executing phpcbf!'
-	exit 1
```

Comment:
Opsie, I was having issues with phpcbf locally and had to this for the task to finish. I'll revert this change.